pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/MhloLegalizeUtils.cpp

```diff
@@ -20,21 +20,41 @@
 
 #include <mlir-hlo/Dialect/mhlo/IR/hlo_ops.h> // from tf repo
 
+#include <iostream>
 #include "torch-mlir/Conversion/TorchToMhlo/MhloLegalizeUtils.h"
 
 namespace mlir {
 namespace mhlo {
 
+std::vector<int64_t> rangeIndices(int64_t min, int64_t max) {
+  std::vector<int64_t> range;
+  for (int64_t k = min; k < max; ++k) {
+    range.push_back(k);
```

Comment:
1. is this identical to `std::iota`?
2. nit: `range.reserve`

Reply:
Change to `std::iota`